packages/Symfony/DepedencyInjection/Compiler/EcotoneCompilerPass.php

```
@@ -61,11 +61,11 @@ public static function getMessagingConfiguration(ContainerInterface $container,
         /** @TODO Ecotone 2.0 use ServiceContext to configure Symfony */
         $serviceConfiguration = ServiceConfiguration::createWithDefaults()
             ->withEnvironment($container->getParameter('kernel.environment'))
-            ->withFailFast($container->getParameter('kernel.environment') === 'prod' ? false : $container->getParameter(self::FAIL_FAST_CONFIG))
+            ->withFailFast(in_array($container->getParameter('kernel.environment'), ['prod', 'production']) ? false : $container->getParameter(self::FAIL_FAST_CONFIG))
```

hmm, does it mean that if I want this "fail fast" behavior in production for some reason, I cannot enable it explicitly in the config? I know it was introduced earlier and this PR just extends on existing behavior, but anyway. Perhaps generating a default config for prod environment with this set to false would suffice?

---

Sure we can make it default only if was not set up. 
Would you be willing to contribute that after this PR is merged?

---

Yeah, I can do this change myself